firefox-ios/Client/Application/RemoteSettingsUtil.swift

```diff
@@ -0,0 +1,248 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import MozillaAppServices
+import Common
+import Shared
+
+enum RemoteCollection: String {
+    case searchTelemetry = "search-telemetry-v2"
+}
+
+enum Remotebucket: String {
+    case defaultBucket = "main"
+}
+
+enum RemoteSettingsUtilError: Error {
+    case decodingError
+    case fetchError(Error)
+}
+
+extension RemoteSettingsRecord: Codable {
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(id, forKey: .id)
+        try container.encode(lastModified, forKey: .lastModified)
+        try container.encode(deleted, forKey: .deleted)
+        try container.encode(attachment, forKey: .attachment)
+        try container.encode(fields, forKey: .fields)
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
```

Comment:
A temporary solution to save the remote settings record as I was testing locally and it seems Application Services does not cache these?

@linabutler correct me if this is not the case

Reply:
That's right, the current Application Services Remote Settings doesn't persist downloaded records (or attachments).